pyanaconda/modules/payloads/source/factory.py

```diff
@@ -35,12 +35,18 @@ def create_source(source_type: SourceType):
         if source_type == SourceType.LIVE_OS_IMAGE:
             from pyanaconda.modules.payloads.source.live_os.live_os import LiveOSSourceModule
             return LiveOSSourceModule()
+
         elif source_type == SourceType.CDROM:
             from pyanaconda.modules.payloads.source.cdrom.cdrom import CdromSourceModule
             return CdromSourceModule()
 
-        if source_type == SourceType.HMC:
+        elif source_type == SourceType.HMC:
             from pyanaconda.modules.payloads.source.hmc.hmc import HMCSourceModule
             return HMCSourceModule()
 
+        elif source_type == SourceType.REPO_FILES:
```

Comment:
`elif-elif` should not have an empty space between them.

Reply:
Fixed!